viewer/src/utilities/networking/HttpHeadersProvider.ts

```diff
@@ -2,7 +2,7 @@
  * Copyright 2020 Cognite AS
  */
 
-import { HttpHeaders } from '@cognite/sdk/dist/src/utils/http/basicHttpClient';
+export type HttpHeaders = { [key: string]: string };
```

Comment:
https://github.com/cognitedata/cognite-sdk-js/blob/a569720c188d3413f46b997f1d17b6efaf7ff65f/packages/core/src/index.ts

Should we just add sdk-core into dev deps and import HttpHeaders from it?

Reply:
I'm not sure - the reason I switched away from importing was that I thought that it might break compatibility rules, i.e. that internal files might be moved around. Or is it something that is public and that we can consider stable?

Reply:
I guess it's public and should be just available directly from sdk-core as `import { HttprHeaders } from '@cognite/sdk-core'`

Reply:
Oh, is sdk-core something new? Will update